db/migrate/20220223231534_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
```

Comment:
- [ ] Let's make sure that we are following the Ruby naming convention. For example, commentscounter, should be `comments_counter`

Reply:
done